app/src/main/java/com/muzima/view/SHRObservationsDataActivity.java

```diff
@@ -55,6 +59,14 @@ protected void onCreate(Bundle savedInstanceState) {
         initPagerIndicator();
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        if (!isSHREnable()){
```

Comment:
Hi @rezita, I think this should be isSHREnabled

Reply:
Hello @benamonya, yes you are right. I changed it. Thanks!